packages/neuron-wallet/src/database/chain/ormconfig.ts

```diff
@@ -26,7 +29,7 @@ const connectOptions = async (genesisBlockHash: string): Promise<SqliteConnectio
   const database = env.isTestMode ? ':memory:' : dbPath(genesisBlockHash)
 
   const logging: boolean | ('query' | 'schema' | 'error' | 'warn' | 'info' | 'log' | 'migration')[] =
-    process.env.SHOW_CHAIN_DB_LOG && (env.isDevMode || env.isTestMode) ? true : ['warn', 'error']
+    (env.isDevMode || env.isTestMode) ? ['warn', 'error', 'log', 'info', 'schema', 'migration'] : ['warn', 'error']
```

Comment:
Default to show logs except `query` in dev and test mode?

Reply:
Yes.

Reply:
Combined with `maxQueryExecutionTime: 30` in `ormconfig.ts`.

Reply:
OK, I means will see long logs when running in dev / test mode.